pkg/volume/plugins.go

```diff
@@ -169,6 +169,19 @@ func (spec *Spec) Name() string {
 // The binary should still use strong typing for this value when binding CLI values before they are passed as strings
 // in OtherAttributes.
 type VolumeConfig struct {
+	// RecyclerDefaultPod is the default pod used to scrub a persistent volume clean after its release.
+	// The default scrubber supplied by the system is a simple "rm -rf /* /.*" of a volume.
+	RecyclerDefaultPod *api.Pod
+
+	// RecyclerMinimumTimeout is the minimum amount of time in seconds for the scrub pod's ActiveDeadlineSeconds attribute.
+	// Added to the minimum timeout is the increment per Gi of capacity.
+	RecyclerMinimumTimeout int
+
+	// RecyclerTimeoutIncrement is the number of seconds added to the scrub pod's ActiveDeadlineSeconds for each
+	// Gi of capacity in the persistent volume.
+	// Example: 5Gi volume x 30s increment = 150s + 30s minimum = 180s ActiveDeadlineSeconds for scrub pod
+	RecyclerTimeoutIncrement int
```

Comment:
Having these attributes per PV provides great flexibility (because it's plausible that performance can differ between NFS providers on different hardware with faster switches, etc.) but that then makes config part of the API.  It was noted previously (see Clayton's comments from mid-July) that this wasn't an API thing but a config-that's-passed-down thing.

This implementation of VolumeConfig means 1 value per plugin, but that can be configured to the lowest common denominator in the user's diverse NFS storage infrastructure.